src/index.ts

```diff
@@ -14,6 +15,9 @@ appInsights.setup(clientKey)
   .setUseDiskRetryCaching(true)
   .setSendLiveMetrics(process.env.AI_ENABLE_LIVEMETRICS === 'true') // Default to disabled: https://github.com/microsoft/ApplicationInsights-node.js/issues/615
   .setDistributedTracingMode(appInsights.DistributedTracingModes.AI_AND_W3C)
+const maxSockets = parseInt(process.env.AI_MAX_SOCKETS || '20')
```

Comment:
defaulting the maxSockets to 20 could easily cause latency and timeouts. Lets do 300 to be safe
```suggestion
const maxSockets = parseInt(process.env.AI_MAX_SOCKETS || '300')
```

Reply:
@jmills-gopuff was of the opinion that it should be a lot lower.  I'll ask about it on slack

Reply:
I'd rather just get this out the door TBH.  300 (and 150) are really large numbers for a single threaded node app, but we aren't going to ever hit these numbers either, so I'm fine with setting it to 150.

Reply:
Sure, You can always override the default, like I said, setting them too low as Josh is suggesting can introduce latency. 
This is pretty easy to test locally.